lib/iscsi-command.c

```diff
@@ -327,6 +327,13 @@ static void parse_sense_desc_info(struct scsi_sense *sense,
 	sense->information = scsi_get_uint64(&inf[0]);
 }
 
+static void parse_sense_fixed_info(struct scsi_sense *sense,
```

Comment:
Please inline this function. I don't think it's worth it to introduce a new function for these two statements.